extensions/blocks/image-compare/image-compare.php

```diff
@@ -58,9 +58,9 @@ function render_amp( $attr ) {
 	$img_after  = $attr['imageAfter'];
 
 	return sprintf(
-		'<amp-image-slider layout="responsive" width="%1$d" height="%2$d"> <amp-img id="%3$d" src="%4$s" alt="%5$s" layout="fill"></amp-img> <amp-img id="%6$d" src="%7$s" alt="%8$s" layout="fill"></amp-img></amp-image-slider>',
-		absint( $img_before['width'] ),
-		absint( $img_before['height'] ),
+		'<amp-image-slider layout="responsive"%1$s%2$s> <amp-img id="%3$d" src="%4$s" alt="%5$s" layout="fill"></amp-img> <amp-img id="%6$d" src="%7$s" alt="%8$s" layout="fill"></amp-img></amp-image-slider>',
+		! empty( $img_before['width'] ) ? ' width="' . absint( $img_before['width'] ) . '"' : '',
+		! empty( $img_before['height'] ) ? ' height="' . absint( $img_before['height'] ) . '"' : '',
```

Comment:
This caused a regression: #16093.

Note: In AMP if the `width`/`height` are not available, instead of omitting the attributes (since they are required in AMP), the solution is to provide fallback `width` and `height`. See proposed fix in #17253.